packages/story-editor/src/components/canvas/rightClickMenu.js

```diff
@@ -42,6 +42,45 @@ const RightClickMenuContainer = styled.div`
   z-index: 9999;
 `;
 
+const ButtonWrapper = styled.div`
+  position: relative;
+`;
+
+const SubMenuWrapper = styled(ContextMenuComponents.PlainMenu)`
+  width: 230px;
+  position: absolute;
+  left: calc(100% + 2px);
+  top: -9px;
+`;
+
+const LabelWrapper = styled.span`
+  width: calc(100% + 14px);
+  display: inline-block;
+  position: relative;
+  margin-left: -14px;
+  svg {
+    width: 32px;
+    position: absolute;
+    top: -25%;
+  }
+  span {
+    padding-left: 32px;
+    font-size: 12px;
+    display: inline-block;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+`;
+
+const SuffixIconWrapper = styled.div`
+  svg {
+    width: 32px;
+    color: ${({ theme }) => theme.colors.fg.secondary};
+    margin: -12px 0 !important;
+  }
+`;
```

Comment:
🤔 If we're to follow nesting context menus like the comment below then all of these styles shouldn't be necessary

There's a component defined in the `contextMenu` folder called `Label` (exported as `MenuLabel`). Styling this may help position and size icons in the sub menu. Otherwise you could style the `Button` directly to deal with icons as children.